blocks/init/src/Blocks/wrapper/components/wrapper-options.js

```diff
@@ -272,6 +276,21 @@ export const WrapperOptions = ({ attributes, setAttributes }) => {
 						/>
 					}
 
+					<Section
+						icon={icons.code}
+						label={__('Wrapper tag', 'eightshift-frontend-libs')}
+						showIf={showWrapperTag}
+					>
+						<OptionSelector
+							options={wrapperTagOptions}
+							value={wrapperTag}
+							onChange={(value) => setAttributes({ wrapperTag: value })}
+							additionalButtonClass='es-v-spaced es-w-16 es-text-3 es-content-center'
+							noBottomSpacing={getWrapperUseOptionName() === 'off'}
+							alignment='center'
+						/>
+					</Section>
+
```

Comment:
```suggestion
					{wrapperUse && showWrapperTag &&
						<OptionSelector
							label={__('Wrapper tag', 'eightshift-frontend-libs')}
							icon={icons.code}
							options={wrapperTagOptions}
							value={wrapperTag}
							onChange={(value) => setAttributes({ wrapperTag: value })}
							additionalButtonClass='es-v-spaced es-w-16 es-text-3 es-content-center'
							alignment='center'
						/>
					}

```
Formatting might be wonky, but TL;DR
- don't need the Section 😄 
- `noBottomSpacing` is not a concern if you can hide the whole control when the wrapper is off